stable/xray/templates/xray-indexer-statefulset.yaml

```diff
@@ -27,6 +27,9 @@ spec:
         component: {{ .Values.indexer.name }}
       annotations:
         checksum/config: {{ include (print .Template.BasePath "/xray-setup-conf.yaml") . | sha256sum }}
+        {{- with .Values.analysis.annotations }}
```

Comment:
This should be indexer.annotations
Please use the indent function

Reply:
Resolved!